SHARDING.md

```diff
@@ -307,13 +325,13 @@ have picked up the latest TrustedRoot.
 if they are after this date. This must be after the start of the new shard. It doesn't have to be
 overly precise, and can be updated in a future TUF signing event.
 
-### Retrieve shard public key
+### Get shard public key
 
-Look at the service logs to find the public key. This is the easiest way to retrieve
-the public key, as there is no CLI tool to convert a Tink keyset into a DER-encoded public key.
+You should have the log public key saved in `public.pem` from when you generated the key.
+If you don't have the public key, look at the service logs to find the public key, which
+is logged on service startup.
 
-The key will be printed as a PEM-encoded public key. Remove the PEM header and footer and remove
-all newline characters to get the DER-encoded public key.
+The key is PEM-encoded. Remove the PEM header and footer and remove all newline characters to get the DER-encoded public key.
```

Comment:
nit: I think calling it DER encoded is a bit misleading -- it's not binary like DER is supposed to be

Reply:
Thanks, updated to state this is base64-encoded PKIX, which is more precise.